packages/@jsii/java-runtime/project/src/test/java/software/amazon/jsii/JsiiRuntimeTest.java

```diff
@@ -0,0 +1,73 @@
+package software.amazon.jsii;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Collections;
+
+public final class JsiiRuntimeTest {
+    @Test
+    public void withNoCustomization() {
+        final JsiiRuntime runtime = new JsiiRuntime(null, null);
+        runtime.getClient().createObject("Object", Collections.emptyList(), Collections.emptyList(), Collections.emptyList());
+        runtime.terminate();
```

Comment:
[nit] `runtime` could be a `JsiiRuntimeTest` class member and this line fitted into https://junit.org/junit5/docs/5.0.2/api/org/junit/jupiter/api/AfterEach.html